masschallenge/impact-api
# AC-7816 Include office hours meeting info in Add to Cal

### Changes Introduced in [AC-7816](https://masschallenge.atlassian.net/browse/AC-7816):
- Include office hours links
### Testing:
- Check out this branch
- Masquerade as a mentor and a finalist
- Change their emails to your own before scheduling an office hour.
- Schedule an office hour and reserve it. Make sure you fill something out in the "meeting info" field
- Check the resulting email and the calendar links. Notice that the specified fields are included
